analyticConfigurations/computed_metric-route53.json

```diff
@@ -0,0 +1,19 @@
+{
+  "analyticConfiguration": {
+    "metrics": [
+      {
+        "match": "aws.route53.healthecheckstatus.min",
```

Comment:
Does this metric measure only whether at least one region is unhealthy? If so, would it be more useful to calculate the sum of unhealthy regions instead?

Reply:
I believe this metric indicates if any regions are unhealthy. If 1 all regions are healthy, otherwise something is unhealthy.

Reply:
Yes, it works in this way. What would be better - to leave it as is or calculate the sum of unhealthy regions instead?

Reply:
Done. Changed the computed metric expression and renamed it to  aws.route53.healthcheckstatus.sumunhealthy

Reply:
Remind me where we ended up on this metric? Which of these did we decide we could/should have?

- min health across all regions
- sum unhealthy regions
- percentage unhealthy regions

Reply:
We ended up here: https://metricly.slack.com/archives/C1Y1R60MP/p1544642452143600?thread_ts=1544632419.136300&cid=C1Y1R60MP
I renamed aws.route53.healthcheckstatus.sumunhealthy to aws.route53.healthcheckstatus.all and changed calculation to the percentage in the yesterday's commits. After that I did an additional research and it looks like the CloudWatch provides global/accumulated data under us-east-1 metric.
I can quickly add min and sum if reuired, but maybe we can temporarily disable the computed metric until AWS/CloudWatch starts providing route53.healthcheckstatus data per each region?